manifests/plugin/deb.pp

```diff
@@ -1,14 +1,14 @@
 # @summary Pulp Deb plugin
-# @param use_pulp2_content_route
```

Comment:
Should we also keep this @ekohl ?

Reply:
I'd imagine so. These are endpoints that Pulp 2 users can use continue using.